index.js

```diff
@@ -28,10 +28,10 @@ function svg2ttf(svgString, options) {
   font.copyright = options.copyright || svgFont.metadata;
   font.description = options.description || 'Generated by svg2ttf from Fontello project.';
   font.url = options.url || 'http://fontello.com';
-  font.sfntNames.push({ id: 2, value: options.subfamilyname || svgFont.subfamilyName || 'Regular' }); // subfamily name
-  font.sfntNames.push({ id: 4, value: options.fullname || svgFont.id }); // full name
+  font.sfntNames.push({ id: 2, value: (options.subfamilyname || svgFont.subfamilyName || 'Regular').trim() }); // subfamily name
```

Comment:
I don't like idea of auto-trim. IMO that's a garbage for artificial case.

Reply:
I prefer svg2ttf to filter out these extraneous whitespace characters in order to produce a cleaner font file. Similar processing is also done in ufo2ft, of course we can only consider familyName and styleName.

https://github.com/googlefonts/ufo2ft/blob/a5267d135d5a8dba7e6a5d3ac44139afa6159429/Lib/ufo2ft/fontInfoData.py#L74

Reply:
Sorry, i do not agree, IMO tool should write data exactly as been passed. IMO if you need trimming, it should be filtered on upper app level.

Reply:
I also changed my mind, and now I agree with you.